examples/tutorials/Gpaw_cell_tutorial.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Add some more references to the literature on DFT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>